utils/hare-fetch-fids

```diff
@@ -375,8 +406,12 @@ def main(argv=None):
             else:
                 print(simplejson.dumps(svcs, indent=2, for_json=True))
 
+    except HAConsistencyException as err:
+        print(f"Could not read data from KV store '{err}'")
```

Comment:
Let's log this to log file, instead of printing on console. I think it would be good to save the log file path in consul kv so that it can be fetched from consul directly, by default we can use `/var/log/seagate/hare/hctl.log` for all hctl commands.